cpp/src/IceStorm/msbuild/icestormadmin/icestormadmin.vcxproj

```diff
@@ -84,9 +84,12 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Label="IceBuilder">
     <SliceCompile>
-      <IncludeDirectories>$(IceSrcRootDir)\src\;%(IncludeDirectories)</IncludeDirectories>
-      <HeaderOutputDir>$(Platform)\$(Configuration)\IceStorm</HeaderOutputDir>
-      <BaseDirectoryForGeneratedInclude>IceStorm</BaseDirectoryForGeneratedInclude>
+      <IncludeDirectories>
```

Comment:
I cleared these settings with the GUI. Not sure why it results in empty definitions.

Reply:
There is an option to set to inherit, which gets rid of it.

Reply:
This works, but it's oddly not persistent. After you save, you can't tell the difference between inherit and blank.

Reply:
Updated the project files.

The difference is once you've selected "Inherit from project defaults...", you don't see this option anymore.